tests/unit/source_code/linters/test_python_imports.py

```diff
@@ -216,22 +147,27 @@ def test_args_count(code, expected):
 name = "xyz"
 dbutils.notebook.run(name)
 """,
-            "xyz",
-        )
+            ["xyz"],
+        ),
+        (
+            """
+name = "xyz" + "-" + "abc"
+dbutils.notebook.run(name)
+""",
+            ["xyz-abc"],
+        ),
+        (
+            """
+names = ["abc", "xyz"]
```

Comment:
can we already do 

```
def foo(): return "bar"
name = foo()
dbutils.notebook.run(name)
```

or does it require building a small type-aware interpreter?

Reply:
we can! added corresponding test in `test_infers_dbutils_notebook_run_dynamic_value`

Reply:
Nice!